### ISSUE TITLE
Ruby Sass is deprecated and will be unmaintained as of 26 March 2019.

### ISSUE THREAD

**Issue by tmartyny** [twbs/bootstrap-sass#1171]
Hello! When I bundle update, I'm getting this error:
```
Ruby Sass is deprecated and will be unmaintained as of 26 March 2019.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  http://sass.logdown.com/posts/7081811
```

**Comment by fernandoluizao** [comment]
I think it's only needed to change to sassc here:

https://github.com/twbs/bootstrap-sass/blob/master/bootstrap-sass.gemspec#L14

And release a new version to rubygems.

**Comment by glebm** [comment]
This has already been fixed in the `next` branch. Until it is released (along with v3.4 or before EOY), you can use the `next` branch like this:

```ruby
gem 'bootstrap-sass', git: 'https://github.com/twbs/bootstrap-sass', branch: 'next'
```
